site/docs/foundations/size.mdx

```diff
@@ -5,68 +5,70 @@ sidebar:
 
 ## Size tokens
 
-| Token                    | Dimension        | High Density (HD) | Medium Density (MD) | Low Density (LD) | Touch Density (TD) |
-| ------------------------ | ---------------- | ----------------- | ------------------- | ---------------- | ------------------ |
-| `--salt-size-base`       | Height           | 20px              | 28px                | 36px             | 44px               |
-| `--salt-size-selectable` | Height and width | 12px              | 14px                | 16px             | 18px               |
-| `--salt-size-icon`       | Height and width | 12px              | 12px                | 14px             | 16px               |
-| `--salt-size-adornment`  | Height and width | 6px               | 8px                 | 10px             | 12px               |
-| `--salt-size-bar`        | Height or width  | 2px               | 4px                 | 6px              | 8px                |
-| `--salt-size-indicator`  | Height or width  | 1px               | 2px                 | 3px              | 4px                |
-| `--salt-size-border`     | Height or width  | 1px               | 1px                 | 1px              | 1px                |
+Consistent use of size tokens ensures uniformity in component dimensions, enhancing the visual harmony and functionality of your application's design. Each size token responds to density, ensuring components remain aligned as the density is changed.
+
+| Token                    | Dimension        | High Density (HD) (px) | Medium Density (MD) (px) | Low Density (LD) (px) | Touch Density (TD) (px) |
+| ------------------------ | ---------------- | ---------------------- | ------------------------ | --------------------- | ----------------------- |
+| `--salt-size-base`       | Height only      | 20                     | 28                       | 36                    | 44                      |
+| `--salt-size-selectable` | Height and width | 12                     | 14                       | 16                    | 18                      |
+| `--salt-size-icon`       | Height and width | 12                     | 12                       | 14                    | 16                      |
+| `--salt-size-adornment`  | Height and width | 6                      | 8                        | 10                    | 12                      |
+| `--salt-size-bar`        | Height or width  | 2                      | 4                        | 6                     | 8                       |
+| `--salt-size-indicator`  | Height or width  | 1                      | 2                        | 3                     | 4                       |
+| `--salt-size-border`     | Height or width  | 1                      | 1                        | 1                     | 1                       |
```

Comment:
`--salt-size-border` shouldn't be in this table. `--salt-size-bar-strong` is missing.